packages/@sanity/cli/src/__telemetry__/cli.telemetry.ts

```diff
@@ -0,0 +1,29 @@
+import {defineTrace} from '@sanity/telemetry'
+
+interface CLITraceData {
+  /**
+   * Command flags, without the core options (help, debug, version etc)
+   */
+  groupOrCommand: string
+
+  /**
+   * Command arguments, eg any arguments after `sanity <command>` (no flags)
+   */
+  commandArguments: string[]
+
+  /**
+   * Arguments after the ended argument list (--)
+   */
+  extraArguments: string[]
+  coreOptions: {
+    help: boolean
+    debug: boolean
+    version: boolean
+  }
+}
+
+export const CliCommand = defineTrace<CLITraceData>({
+  name: 'cliCommand',
```

Comment:
@bjoerge This should still be "CLI Command Executed" right?

Reply:
Yes!